Voluntary Exit for Capella/Shanghai updates (#704)
prysmaticlabs/documentation: website/docs/wallet/exiting-a-validator.md

import {HeaderBadgesWidget} from '@site/src/components/HeaderBadgesWidget.js';

<HeaderBadgesWidget  commaDelimitedContributors="James" lastVerifiedDateString="February 3rd, 2023" lastVerifiedVersionString="v3.2.0"/>

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> Voluntarily exiting your validator from the Ethereum network is a one time command using the [prysmctl tool](../prysm-usage/prysmctl.md). **note:** previously uses a command under the validator client, and can still be accessed that way.

Reviewer: ```suggestion
Voluntarily exiting your validator from the Ethereum network is a one-time command using the [prysmctl tool](../prysm-usage/prysmctl.md). **Note:** This command was previously exposed through Prysm's validator client, and can still be accessed that way.
```
